Externals/hidapi/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+project(hidapi)
+
+set(SRCS)
```

Comment:
CMake complained when I tried `add_library(hidapi)`. I just thought of something, I bet it would accept a header-only library `hidapi/hidapi.h` to start with. I'll work on it.